content/docs/guides/crosswalk/kubernetes/introduction/01-architecture.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Production Architecture for Teams
+
+aliases: ["/docs/guides/crosswalk/kubernetes/architecture/"]
+---
+
+Infrastructure misconfiguration accounts for the most significant subset of serious
+production outages in major systems. Some [reasonable estimates][post-mortems] put the fraction in
+the range of 40-50%.
+
+Many of these failure modes are trivially preventable. The primary objective of this reference
+architecture is to create sensible defaults that make many of these issues either impossible, or
+dramatically less likely.
+
+**Modern Infrastructure-as-Code tools, such as Pulumi, are effective means of accomplishing
+this goal.**
+
+These tools allow engineering teams to share specifications for what their
+infrastructure should look like. These specifications can be executed, allowing teams to
+reliably provision and manage infastructure. Changes to infrastructure can be audited as part of
+code review. And they allow teams to detect drift.
+
+But even the most powerful tools don't solve problems -- teams do. And doing
+so requires the organizational discipline to use such tools effectively.
+
+This architecture is meant to show how infrastructure-as-code tools can be used in a team setting.
+It is meant to answer questions like:
+
+* **Security:** Who has access to what, and how are things ensured?
+* **Governance:** How do we ensure the blast radius of changes is as small as possible?
+* **Engineering:**  How do we automate this with CI/CD?
+
+## Production Infrastructure-as-Code
```

Comment:
Nit, we tend to use "infrastructure as code," no hyphens. It would be good to be consistent. I changed a few above but will just comment here as it's probably easier to search and replace.

Reply:
Done